packages/app/screens/feed/create group form/CreateGroupFormScreen1.js

```diff
@@ -0,0 +1,236 @@
+import React, { useEffect, useState, useContext } from "react";
+import {
+  View,
+  Text,
+  TextInput,
+  KeyboardAvoidingView,
+  TouchableWithoutFeedback,
+  Keyboard,
+  Image,
+  StyleSheet
+} from "react-native";
+import Button from "@app/components/Button";
+import * as ImagePicker from "expo-image-picker";
+import { GroupContext } from "@app/utils/GroupContext";
+import { ScrollView } from "react-native-gesture-handler";
+import { styles } from "./CreateGroupFormStyles";
+
+function CreateGroupFormScreen1({ navigation }) {
+
+  // 1st SCREEN INPUTS
+  const {
+    title, setTitle, 
+    handler, setHandler,
+    description, setDescription,
+    banner, setBanner,
+    icon, setIcon,
+    resetForm } = useContext(GroupContext);
+
+  // Input Validation
+  const [titleError, setTitleError] = useState("");
+  const [handlerError, setHandlerError] = useState("");
+  const [descriptionError, setDescriptionError] = useState("");
+  const [bannerError, setBannerError] = useState("");
+  const [iconError, setIconError] = useState("");
+
+  useEffect(() => {
+    handleInputValidationScreen1();
+  }, [title, handler, description, banner, icon]);
+
+  function handleInputValidationScreen1() {
+    let isValid = true;
+
+    const titleRegex = /^(?![a-zA-Z0-9]+$).*$/;
+    const handlerRegex = /^(?!(\w{1,15})$).*$/;
+
+    if (title.trim() === ``) {
+      setTitleError(`Please enter a title`);
+      isValid = false;
+    } else if (titleRegex.test(title)) {
+      setTitleError(`Can only contain letters, numbers, and underscores.`);
+      isValid = false;
+    } else {
+      setTitleError(``);
+    }
+
+    if (handler.trim() === ``) {
+      setHandlerError(`Please enter a handler`);
+      isValid = false;
+    } else if (handlerRegex.test(handler)) {
+      setHandlerError(`Can only contain letters, numbers, and underscores.`);
+      isValid = false;
+    } else {
+      setHandlerError(``);
+    }
+
+    if (description.trim() === ``) {
+      console.log(0);
+      setDescriptionError(`Please enter a description`);
+      isValid = false;
+    } else {
+      setDescriptionError(``);
+    }
+
+    if (banner.trim() === ``) {
+      setBannerError(`Please pick a banner image`);
+      isValid = false;
+    } else {
+      setBannerError(``);
+    }
+
+    if (icon.trim() === ``) {
+      setIconError(`Please pick a icon image`);
+      isValid = false;
+    } else {
+      setIconError(``);
+    }
+
+    // isValid = true; //for testing
+    return isValid;
+  }
```

Comment:
Red Asterisks should appear when you try to press "next" button (and not right away). They could also maybe go away as you type or select an image.